easybuild/easyconfigs/r/R-bundle/R-bundle-20150605-intel-2015a-R-3.1.3-pbd.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'Bundle'
+
+name = 'R-bundle'
+version = '20150605'
+bundlename = 'pbd'
+rver = '3.1.3'
+versionsuffix = '-R-%s-%s' % (rver, bundlename)
+
+homepage = 'http://www.r-project.org/'
+description = """R is a free software environment for statistical computing and graphics."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+# these are extensions for R
+exts_defaultclass = 'RPackage'
+exts_filter = ("R -q --no-save", "library(%(ext_name)s)")
+
+dependencies = [
+    ('R', rver),
+]
+
+name_tmpl = '%(name)s_%(version)s.tar.gz'
+ext_options = {
+    'source_urls': [
+        'http://cran.r-project.org/src/contrib/Archive/%(name)s',  # package archive
+        'http://cran.r-project.org/src/contrib/',  # current version of packages
+        'http://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
+    ],
+    'source_tmpl': name_tmpl,
+}
+
+# !! order of packages is important !!
+exts_list = [
+    ('pbdMPI', '0.2-5', dict(ext_options.items() + [('configureargs', ['--with-mpi-type=INTELMPI'])])),
```

Comment:
the `configureargs` shouldn't be there, since the pdbMPI easyblock in https://github.com/hpcugent/easybuild-easyblocks/pull/620 takes care of that?